src/components/data-management/ProjectCard.jsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  Card, Descriptions,
+} from 'antd';
+import { blue } from '@ant-design/colors';
+import { useSelector } from 'react-redux';
+import EditableField from '../EditableField';
+import PrettyTime from '../PrettyTime';
+
+const { Item } = Descriptions;
+
+const activeProjectStyle = {
+  backgroundColor: blue[0],
+  cursor: 'pointer',
+  border: `2px solid ${blue.primary}`,
+};
+
+const ProjectCard = (props) => {
```

Comment:
Can you settle on one convention and make sure that all component names follow it? Are we going to use Project**s** for all component names that are about multiple projects? Then rename this and every other component to be `Projects<blablabal>`. If not, then again make sure all components are consistent.

Reply:
I was differentiating it depending on whether the component relates to a single project (e.g. **Project**Card) or multiple projects (e.g. **Projects**List). However, in the interest of clarity, I think it's better to use **Project** for the prefix of all components.

Reply:
Ok, so there was a reasoning behind that differentiating, good, this is all I was asking for. Then I would leave it to `ProjectCard` if it applies to a single project

Reply:
On 2nd thought, I think the current naming is good. This is called `ProjectCard` because it contains only 1 project, not multiple project.

`ProjectsList` is a list of projects
`ProjectsListContainer` is the container for `ProjectsList`.